enterprise/tpa_pipeline.py

```diff
@@ -20,24 +20,41 @@
 except ImportError:
     Registry = None
 
+try:
+    from third_party_auth.pipeline import get as get_pipeline_partial
+except ImportError:
+    get_pipeline_partial = None
+
+
+def verify_third_party_auth_dependencies():
+    """
+    Ensure that all necessary third_party_auth dependencies are present.
+    """
+    third_party_dependencies = (
+        Registry,
+        get_pipeline_partial,
+    )
+
+    if any(third_party_dependency is None for third_party_dependency in third_party_dependencies):
+        raise NotConnectedToEdX(
+            _("This package must be installed in an EdX environment to look up third-party auth dependencies.")
```

Comment:
If it's too much scope to rename the actual exception for this particular PR, then we can just update this message to say "Open edX" and log a tech debt story to fix the exception name.